package.json

```diff
@@ -42,6 +42,8 @@
   },
   "dependencies": {
     "concurrently": "^8.2.2",
+    "graphql": "^16.8.1",
+    "graphql-request": "^6.1.0",
```

Comment:
@boudra Is there a better way of doing this instead of adding it to the root package.json ?  One option would be throwing it into the grant-explorer package.json

Reply:
made change based off suggestion